.github/workflows/GC.yml

```diff
@@ -18,4 +19,4 @@ jobs:
       - name: Install dependencies
         run: dotnet restore
       - name: Build
-        run: dotnet build --configuration Release --no-restore
+        run: dotnet build --configuration Release --no-restore
```

Comment:
Please add new line at the end of the file

Reply:
done